public/js/stores/ArticleStore.js

```diff
@@ -5,18 +5,34 @@ class ArticleStore extends EventEmitter {
     constructor() {
         super()
         this.articles = []
+        this.sort = 'NEW'
```

Comment:
Alright we'll stick with this PR for now, and wait on the `trending` to be default